dist/morphlex.js

```diff
@@ -109,9 +110,6 @@ function morphChildElement(child, ref, parent, idMap) {
 		morphNodes(nextMatchByTagName, ref, idMap);
 	} else child.replaceWith(ref.cloneNode(true));
 }
-// We cannot use `instanceof` when nodes might be from different documents,
-// so we use type guards instead. This keeps TypeScript happy, while doing
-// the necessary checks at runtime.
```

Comment:
I assume this comment is being removed because it's associated with the method overload in the TypeScript, which doesn't get output here.